samples/sampler/src/ThemeEnvironment/ThemeEnvironment.js

```diff
@@ -224,15 +226,32 @@ const StorybookDecorator = (story, config) => {
 	globals.nightMode = boolean('night mode', Config, globals.nightMode);
 
 	let skinKnobs = {};
-	let {accent, highlight} = {};
+	let isSkinChanged = false;
+
 	if (!globals.allSkins) {
-		globals.skin = select('skin', skins, Config, skinFromURL || globals.skin);
-		skinKnobs.skin = globals.skin;
-		globals.defaultSkinStyles = boolean('default skin styles', Config, globals.defaultSkinStyles);
-		if (globals.defaultSkinStyles) {
-			accent = defaultColors[globals.skin].accent;
-			highlight = defaultColors[globals.skin].highlight;
+		skinKnobs.skin = select('skin', skins, Config, skinFromURL || globals.skin);
+		if (skinKnobs.skin !== globals.skin) {
+			isSkinChanged = true;
 		}
+		globals.skin = skinKnobs.skin;
+		globals.defaultSkinStyles = boolean('default skin styles', Config, globals.defaultSkinStyles);
+	}
+
+	if (globals.defaultSkinStyles || isSkinChanged) {
+		globals.accent = defaultColors[globals.skin].accent;
+		globals.highlight = defaultColors[globals.skin].highlight;
+	}
+
+	if (!globals.defaultSkinStyles) {
+		// NOTE: 'color' function has a bug. It doesn't return a changed value when the value (2nd parameter) is changed. It seems it doesn't reload new value.
+		globals.accent = color('accent', accentFromURL || globals.accent || Config.defaultProps.accent, Config.groupId);
+		globals.highlight = color('highlight', highlightFromURL || globals.highlight || Config.defaultProps.accent, Config.groupId);
```

Comment:
Config.defaultProps.accent -> Config.defaultProps.highlight ?

Reply:
The code does not exist cause reverted.
Thank you for the review :)